hack/common.sh

```diff
@@ -36,9 +37,9 @@ NOOBAA_CSV="$OUTDIR_TEMPLATES/noobaa-csv.yaml"
 ROOK_CSV="$OUTDIR_TEMPLATES/rook-csv.yaml.in"
 OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 
-DEFAULT_IMAGE_REGISTRY="quay.io"
-DEFAULT_REGISTRY_NAMESPACE="ocs-dev"
-DEFAULT_IMAGE_TAG="latest"
+DEFAULT_IMAGE_REGISTRY=$(grep -m 1 IMAGE_REGISTRY $MAKEFILE | sed -e 's/^.*= "//g;s/"//g')
```

Comment:
Instead of doing a grep, I guess passing this information via env variable would be better.

Reply:
I am getting this info from the Makefile by reading through the lines and providing necessary regex to eliminate some garbages. How can I do the same via an environment variable? @ashishranjan738